test.py

```diff
@@ -173,15 +152,15 @@ def parse_args():
 
                         pt_t2 = x1, y1 + 40
                         cv2.putText(frame, '{:.2f}'.format(100*e[0]['confidence']) + ' %', pt_t2, cv2.FONT_HERSHEY_SIMPLEX, 0.5, (0, 0, 255))
+                        if configs['ai']['calc_dist_to_bb']:
```

Comment:
Mind providing a description for what `calc_dist_to_bb` means?

Reply:
calc_dist_to_bb is enabling or disabling depth calculation for bounding box (some models don't have bounding box, or some HW don't have stereo cams

Reply:
Thanks!